core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -652,7 +652,7 @@ class TransactionsTest extends IntegrationTestHarness {
       producer.commitTransaction()
 
       var producerStateEntry =
-        brokers(partitionLeader).logManager.getLog(new TopicPartition(testTopic, 0)).get.producerStateManager.activeProducers.head._2
+        brokers(partitionLeader).logManager.getLog(new TopicPartition(testTopic, 0)).get.producerStateManager.activeProducers.asScala.head._2
```

Comment:
Can we use `.get(0)` instead of `asScala.head`? There's one other similar example in this file.